atom/browser/api/atom_api_app.cc

```diff
@@ -378,9 +378,21 @@ void OnClientCertificateSelected(
     v8::Isolate* isolate,
     std::shared_ptr<content::ClientCertificateDelegate> delegate,
     mate::Arguments* args) {
+  if (args->Length() == 2) {
```

Comment:
We want request to continue with `nullptr` when the callback is called with no arguments or null. Here `callback` has already two bound arguments on c++ side, so when `callback()` is called, the condition will be satisfied.